data/import/foodscapes/README.md

```diff
@@ -0,0 +1,16 @@
+# Creating a SQLite db from Foodscapes CSV data
+
+This should be set up to run in a Docker container: as a first quick step,
+however, the following steps should be all that is needed. The only dependency
+is the `sqlite3` binary.
+
+```
+sudo apt install sqlite3 # or whatever appropriate to install sqlite3 on your OS
+cp /path/to/local/copy/of/foodscapes.csv ./source
+sh foodscapes_csv-to-sqlite.sh
+```
+
+The Foodscapes data will be available as `./dest/foodscapes.db`.
+
+Indexes are currently added to each column: hence a very large file size,
+compared to the source CSV file.
```

Comment:
After creating the sqlite file, I have to copy/move it to `datassette/data` no? If so, maybe note it down here too would be nice

Reply:
right. it's even a bit more complicated than that, so I documented a couple of options in 43f2a90b76cddeee7137fb5fbe918850771b3a4b